docs/spec_draft.md

```diff
@@ -345,6 +346,56 @@ logical operation.
 
 [Back to Ops](#index-of-ops)
 
+## stablehlo.bitcast_convert
+
+### Semantics
+
+Performs an element-wise bitcast operation from `operand` tensor and produces
+a `result` tensor where the bits are reinterpreted with the new element type.
+Bitcast is implemented as a low-level cast, so machines with different
+floating-point representations will give different results (e.g. endianesss).
```

Comment:
floating-point representation(endianesss)

Reply:
Bitcast is implemented as a low-level cast, so machines with different
floating-point representation(e.g. endianesss) will give different results.

This goes after formal semantics